src/Xamarin.Android.Build.Tasks/Microsoft.NET.Workload.Android/WorkloadManifest.in.json

```diff
@@ -24,6 +29,26 @@
       "kind": "sdk",
       "version": "@SDK_PACK_VERSION@"
     },
+    "Microsoft.NET.Runtime.MonoAOTCompiler.Task": {
+      "kind": "sdk",
+      "version": "@MONO_AOT_VERSION@"
+    },
```

Comment:
It's missing runtime packs as well

Reply:
I heard from Mikayla that we should be able to drop runtime packs in  `dotnet/packs` -- If that works, I'll do that in another PR and include them in our installer.

Right now you have to have a `NuGet.config` file to get runtime packs, which is not ideal.

Reply:
What? What do you want to hack the system yourself? I believe they should be included in the manifest as everything else

/cc @mhutch

Reply:
I found out recently that we can't use runtime packs in `dotnet/packs`, see: https://github.com/dotnet/sdk/issues/14044

I tried this out here, but it doesn't seem like it works yet: https://github.com/jonathanpeppers/maui-workload/commit/6bd1b0e80074c2263d0e9e49603b1815b557857c